independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/Dockerfile-layout.include.qute

```diff
@@ -18,31 +18,18 @@
 #
 # Then run the container using :
 #
-# docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/{project.artifact-id}-{type}
+# docker run -i --rm -p 8080:8080 quarkus/{project.artifact-id}-{type}
 #
 ###
-FROM {#insert image /}
-{#insert args /}
+FROM registry.access.redhat.com/ubi8/openjdk-{java.version}-runtime:1.10
+
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
-# Install java and the run-java script
-# Also set up permissions for user `1001`
-RUN microdnf install curl ca-certificates $\{JAVA_PACKAGE} \
-    && microdnf update \
-    && microdnf clean all \
-    && mkdir /deployments \
-    && chown 1001 /deployments \
-    && chmod "g+rwX" /deployments \
-    && chown 1001:root /deployments \
-    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/$\{RUN_JAVA_VERSION}/run-java-sh-$\{RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1001 /deployments/run-java.sh \
-    && chmod 540 /deployments/run-java.sh \
-    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/conf/security/java.security
 
 # Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 {#insert copy /}
 
 EXPOSE 8080
-USER 1001
+USER 185
```

Comment:
Do we still need to set the user here?

Reply:
@cescoffier ?

Reply:
Yes, it's all about security (and openshift guidelines)

Reply:
Will the switch from 1001 cause issues?

Reply:
I have same question as ebullient ?

Reply:
Right -- I think we're flipping over to always use the pre-baked UBI images... which is why the jar file name has to change, and why the user id has to change...

Reply:
yes, exactly.